uvloop/cbhandles.pyx

```diff
@@ -163,7 +163,7 @@ cdef class Handle:
 
         return '<' + ' '.join(info) + '>'
 
-    def cancel(self):
```

Comment:
This is [`Handle.cancel()`](https://docs.python.org/3/library/asyncio-eventloop.html#asyncio.Handle) which doesn't have the `msg` parameter.